src/Rules/FunctionDefinitionCheck.php

```diff
@@ -15,17 +15,21 @@
 class FunctionDefinitionCheck
 {
 
-	const VALID_TYPEHINTS = [
-		'self',
-		'static',
-		'array',
-		'callable',
-		'string',
-		'int',
-		'bool',
-		'float',
-		'void',
-		'iterable',
+	const COMMON_TYPEHINTS = [
```

Comment:
typehint is outdated PHP terminology, since PHP 7 typehints have been renames to just *types* as technically they are no hints.